src/main/scala/day1/DayOneMain.scala

```diff
@@ -0,0 +1,120 @@
+package DayOneMain
+
+object DayOneMain {
+
+  def printInput(input: String) {
+    println(input)
+  }
+
+  def returnString(): String = {
+    return "Returned String"
+  }
+
+  def outputAnyInputToConsole(input: Any): Unit = {
+    println(input)
+  }
+
+  def returnNumOfCharsFromStringEnd(numChars: Int, stringToUse: String): String ={
+    val resultOfStringSplit: String = stringToUse.takeRight(numChars)
+    resultOfStringSplit
```

Comment:
last expression is returned hence instead of having
val resultOfStringSplit: String = stringToUse.takeRight(numChars)
    resultOfStringSpli

you could just have
stringToUse.takeRight(numChars)